src/bijectors/named_bijector.jl

```diff
@@ -0,0 +1,231 @@
+abstract type AbstractNamedBijector <: AbstractBijector end
+
+forward(b::AbstractBijector, x) = (rv = b(x), logabsdetjac = logabsdetjac(b, x))
+
```

Comment:
Isn't this the wrong file for such a generic definition? Or is it supposed to be 
```suggestion
forward(b::AbstractNamedBijector, x) = (rv = b(x), logabsdetjac = logabsdetjac(b, x))

```
?